### PR TITLE
Railway documentation addition.

### PR DESCRIPTION
**Pull request by alphasecio**
### What kind of changes does this PR include?

-   [ ] Fixes or refactors
-   [ ] A new challenge
-   [x] Additional documentation
-   [ ] Something else

#### Description

This PR adds documentation for deploying WrongSecrets to the Railway app hosting platform.

### Checklist:

-   [ ] All the contributions made are solely the work of me and my co-authors
-   [ ] I tested the changes in this PR (if applicable)
-   [ ] I added unit tests to ensure my change works (when change in Java or on front-end code)
-   [ ] I added UI tests to ensure my UI changes work (when change in the overall UI, not needed if just adding a challenge)
-   [ ] The PR passes pre-commit hooks and automated tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +135,11 @@ Want to deploy yourself with Render? Click the button below:
 
 [![Deploy to Render](https://render.com/images/deploy-to-render-button.svg)](https://render.com/deploy?repo=https://github.com/OWASP/wrongsecrets)
 
+### Running these on Railway
+
+If you want to host WrongSecrets on Railway, you can do so by deploying [this one-click template](https://railway.app/new/template/7pnwRj), or by clicking the button below. Railway does not offer an always-free plan anymore, but the free trial is good enough to test-drive this before you decide to upgrade. If you need a step-by-step companion guide, see [this blog post](https://alphasec.io/test-your-secret-management-skills-with-owasp-wrongsecrets/).
```
Comment (commjoen) [inline README.md:140]: Can we host the template code definition on this repo please?

File: README.md [hunk 2]
```diff
@@ -134,6 +135,11 @@ Want to deploy yourself with Render? Click the button below:
...
+
+[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/new/template/7pnwRj)
```
Comment (commjoen) [inline README.md:142]: Can we make this a link instead please? After 5 bucks people need to pay, so I rather advertise it less (okteto and intended are truly free, hence they have buttons ;-))

**Review by commjoen — CHANGES_REQUESTED** [review]
Thank you for your pr! Loving it! Can you have a look at the feedback please?

File: README.md (hunk 1 above)
Comment (commjoen) [reply README.md:140]: ah this is only possible for projects, not templates, apologies!

File: README.md (hunk 2 above)
Comment (commjoen) [inline README.md:142]:
```suggestion
[Deploy on Railway](https://railway.app/new/template/7pnwRj)
```

Comment (alphasecio) [reply README.md]: Certainly, we can use your suggestion.

**Review by commjoen — APPROVED** [review]
Thank you @alphasecio !

**Review by commjoen — APPROVED** [review]
This is perfect

**Pull request merged**